src/libraries/Microsoft.PowerFx.Connectors/EngineExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.IR.Nodes;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    public static class EngineExtensions
+    {
+        // To support tabular connectors, we need to use an IR transform to inject the ServiceProvider at runtime (which contains the HttpClient)
+        public static void EnableTabularConnectors(this Engine engine)
```

Comment:
>static [](http://example.com/codeflow?start=15&length=6)

Can we add an [Obsolete] on this? 
Ie, it's something we're going to change in the future. We're adding to unblock limited use case.

Reply:
??
What would you want to replace it with?

Reply:
Not sure, will discuss in phase 3. 
Ideally, this call would not be needed at all. For example, imagine the IR Translator was smart enough to pull rewriters from the type directly (rather than needing a separate registration). 

The point is:
-  it's not intrinsically necessary 
- but fixing it is tricky 
- it's ok for this PR